backend/.env.example

```
@@ -1,39 +1,31 @@
+# ─────────────────────────────────────────────
+# Copy this file to .env and fill in your values
+#   cp .env.example .env
+# Never commit the .env file — it is in .gitignore
+# ─────────────────────────────────────────────
+
+
+# ── Database ──────────────────────────────────
+DATABASE_HOST=localhost
+DATABASE_PORT=5432
+DATABASE_USER=postgres
+DATABASE_PASSWORD=<your-database-password>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_PASSWORD key should go before the DATABASE_PORT key

(UnorderedKey)